### PR TITLE
WIP: Match the page size with Chromium on arm64

### PR DESCRIPTION
**Pull request by zcbenz**

### CODE REVIEW

File: Dockerfile.arm64v8 [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+ libasound2 \
+ libasound2-dev \
+ libcap-dev \
+ libcups2-dev \
+ libdbus-1-dev \
+ libgconf-2-4 \
+ libgconf2-dev \
+ libgnome-keyring-dev \
+ libgtk2.0-0 \
+ libgtk2.0-dev \
```
Comment (ckerr) [inline Dockerfile.arm64v8]: Are the GTK2 libraries still needed?

Comment (zcbenz) [reply Dockerfile.arm64v8]: I'm not sure, this change comes from https://github.com/electron/electron/pull/12080.

Comment (ckerr) [reply Dockerfile.arm64v8]: No, I don't think it's a problem. I was just wondering if there was a reason to keep GTK2, and "for the same image to work in 1.8" is a good answer :smile:

**Comment by zcbenz** [comment]
This is unlikely the correct fix for the crash, `max-page-size` is a linker flag in Chromium that can not affect our Release build, and arm64 build still crashes with this.

**Pull request closed without merging**